natives.json

```diff
@@ -61256,15 +61256,15 @@
 		"0x5F048334B4A4E774": {
 			"name": "_SET_OBJECT_LIGHT_COLOR",
 			"jhash": "",
-			"comment": "",
+			"comment": "If forceColor is set to true, the color specified by r/g/b will override any light sources on the object.\nIf forceColor is set to false, any light sources on the object will be reset to their original color, and the r/g/b arguments will be ignored.\n\nAs of May 2021, on RPH the hash for this function is actually 0x3b2fd68db5f8331c. Not sure of behavior on other hooks.\n\nInfo updated by PNWParksFan, May 2021",

```

Comment:
Not a fan of having info specific to _insert random hook here_ tbh. If they messed up and have an incorrect mapping then they should either fix it or let the users know about it. Not something we should care about here.